bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/AbstractContainerImplTest.java

```diff
@@ -49,4 +49,11 @@ void testGetDorProperties() {
         assertEquals(null, abstractContainerMock.getDorProperties().get("dorTemplateRef"));
     }
 
+    @Test
+    void testGetValidationExpression() {
+        AbstractContainerImpl abstractContainerMock = Mockito.mock(AbstractContainerImpl.class);
+        Mockito.when(abstractContainerMock.getDorProperties()).thenCallRealMethod();
+        assertEquals(null, abstractContainerMock.getValidationExpression());
```

Comment:
lets have an actual test case where validationExpression is not null in jcr.

Reply:
sure

Reply:
@nit23uec I suppose that would be an IT test case..